File: python/1051_imposto_de_renda.py
a = float(input())

if a <= 2000.00:
    print(f'Isento')
else:
    imposto = 0
    
    if a >= 2000.00:
        imposto += (min(a, 3000.00) - 2000.00) * 0.08
    if a > 3000.00:
        imposto += (min(a, 4500.00) - 3000.00) * 0.18
    if a > 4500.00:
        imposto += (a - 4500.00) * 0.28

    print(f'R$ {imposto:.2f}')

File: c/c99/1 - Iniciante/1051_imposto_de_renda.c
#include <stdio.h>

int main() {
    float salario, imposto = 0.0;
    scanf("%f", &salario);

    if (salario <= 2000.00) {
        printf("Isento\n");
    } else if (salario <= 3000.00) {
        imposto = (salario - 2000.00) * 0.08;
        printf("R$ %.2f\n", imposto);
    } else if (salario <= 4500.00) {
        imposto = (1000.00 * 0.08) + (salario - 3000.00) * 0.18;
        printf("R$ %.2f\n", imposto);
    } else {
        imposto = (1000.00 * 0.08) + (1500.00 * 0.18) + (salario - 4500.00) * 0.28;
        printf("R$ %.2f\n", imposto);
    }

    return 0;
}
